app/app.js

```diff
@@ -218,7 +225,7 @@ function processHeroProfile(){
       profilePartners.html('');
       var partner = '';
       $.each(heroPartners, function (index, key){
-        partner += "<span class='partner'>"+key+"</span>"; 
+        partner += "<span class='partner'>"+key+"</span>, "; 
```

Comment:
I feel like you will have a comma at the end doing that. What you can do is put all your `span` in an array, then do `myArray.join(', ')`.